src/Host/Client/Impl/Host/RHost.cs

```diff
@@ -401,6 +401,15 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator {
                                 _callbacks.DirectoryChanged();
                                 break;
 
+                            case "library":
+                                _callbacks.ViewLibrary();
```

Comment:
Would've been great if we generalize this somehow on the level of the protocol an `RHost`, providing general `Override` command to have a switch inside `RSessionCallback`